lib/close_old_pull_requests.rb

```diff
@@ -24,11 +22,11 @@ def initialize(pull_requests)
     end
 
     def outdated
-      pull_requests.group_by { |pr| [pr[:title], pr[:user][:login]] }.values.map do |pulls|
+      pull_requests.group_by { |pr| [pr[:title], pr[:user][:login]] }.values.flat_map do |pulls|
         pulls = pulls.sort_by { |p| p[:created_at] }.reverse
-        new_pr = PullRequest.new(pulls.first[:number])
-        pulls.drop(1).map { |p| PullRequest.new(p[:number], new_pr) }
-      end.compact.flatten
+        new_pr = pulls.first
+        pulls.drop(1).map { |pull| [pull, new_pr] }
```

Comment:
Yep good idea. Added in https://github.com/everypolitician/close_old_pull_requests/pull/13/commits/80a458301f65d303c93adc832da6064aa7a3fcd6.